modules/installation-vsphere-machines.adoc

```diff
@@ -0,0 +1,135 @@
+/ Module included in the following assemblies:
+//
+// * installing/installing_vsphere/installing-vsphere.adoc
+
+[id="installation-vsphere-machines-{context}"]
+= Creating {op-system-first} machines in vSphere
+
+Before you install a user provisioned infrastructure (UPI) cluster on vSphere,
+you need to create {op-system} machines on a single host for it to use.
+
+.Prerequisites
+
+* Obtain the Ignition config files for your cluster.
+* Have access to an HTTP server that you can access from your computer and that
+the machines that you create can access.
+
+.Procedure
+
+////
+This will work for GA.
+. Obtain the {op-system} images from the
+link:https://access.redhat.com/downloads[Product Downloads] page on the Red
+Hat customer portal.
+////
+
+. Convert the bootstrap, master, and worker Ingintion conf files to Base64
+encoding.
+
+. Upload the Base64-encoded bootstrap Ignition config file that the
+installation program created to your HTTP server. Note the URLs of this file.
++
+You must host the bootstrap Ignition config file because it is too large to
+store in temporary memory.
+
+. Save a secondary Ignition config file for your bootstrap node to your computer.
++
+----
+{
+  "ignition": {
+    "config": {
+      "append": [
+        {
+          "source": "<bootstrap_ignition_config_url>", <1>
+          "verification": {}
+        }
+      ]
+    },
+    "timeouts": {},
+    "version": "2.1.0"
+  },
+  "networkd": {},
+  "passwd": {},
+  "storage": {},
+  "systemd": {}
+}
+----
+<1> Specify the URL of the bootstrap Ignition config file that you hosted.
++
+When you create the Virtual Machine (VM) for the bootstrap machine, you reference
+this this Ignition config file.
+
+. Convert the temporary bootstrap file to Base64 encoding.
+
+. Obtain the {op-system} OVA image:
++
+----
+$ curl -J -L -O https://releases-art-rhcos.svc.ci.openshift.org/art/storage/releases/ootpa/410.8.20190412.1/rhcos-410.8.20190412.1-vmware.ova
```

Comment:
Is this the final recommended way for a user to obtain the OVA? Or do we still have plans to update this a URL that will yield the latest OVA?

Reply:
Probably not, but I have [an issue](https://jira.coreos.com/browse/OSDOCS-399) to confirm download/file locations before GA.